flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -26,26 +26,56 @@ MINIKUBE_START_RETRIES=3
 MINIKUBE_START_BACKOFF=5
 RESULT_HASH="e682ec6622b5e83f2eb614617d5ab2cf"
 
+# If running tests on non-linux os, the kubectl and minikube should be installed manually
+function setup_kubernetes_for_linux {
+    # Download kubectl, which is a requirement for using minikube.
+    if ! [ -x "$(command -v kubectl)" ]; then
+        local version=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
+        curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$version/bin/linux/amd64/kubectl && \
+            chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
I think we cannot assume that we have sudo rights.

Reply:
Ok I see that this was done before like this as well. Hence, there must be sudo rights.

Reply:
Yeah, we have the `sudo: required` in `.travis.yml`. So we will have the sudo permission without providing password.